larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -652,6 +652,18 @@ void EvalOpt(TfLiteContext* context, TfLiteNode* node,
   auto unpacked_output_shape = GetTensorShape(output);
   unpacked_output_shape.SetDim(3, params->channels_out);
 
+  // We set caching to true (only for our bconv op). Doing this allows Ruy to
+  // cache the result of packing the weights matrix, which significantly reduces
+  // benchmark latency, at the cost of slightly increased memory usage. The
+  // increase will be approximately the size of the bitpacked model weights
+  // (plus a small amount of overhead if the kernel layout doesn't divide the
+  // shape of the weights). On our target devices we have many GB of memory
+  // available, and we expect the bitpacked model weights to be at most low
+  // double-digit MB, so this is very unlikely to cause any problems.
+  CpuBackendContext* cpu_backend_context =
+      CpuBackendContext::GetFromContext(context);
+  cpu_backend_context->SetUseCaching(true);
```

Comment:
I haven't looked into the `SetUseCaching` API, but are we sure this only enables it for the bconv op and not for everything? Since the `context` object is shared, so are we not setting a global flag here?
This is fine with me, I'm just curious.

Reply:
That is a good question, you might well be right about that. I'll investigate.

Reply:
Yep you're definitely right, thanks for pointing that out. To be safe, let's cache the set value, set it to true, run our op eval, and reset it to the original value at the end of inference.